### PR TITLE
Add the entire survey to the navigation drawer

### LINKED ISSUE #652: Add the entire survey to the navigation drawer
## Kind (Required)

- Improvement

 ## Overview (Required)

- We want to add entire survey("全体アンケート") navigation drawer item to Navigation Drawer
- Since the URL has not yet been determined, please access google.com when you press the button. 🙏 

 ## Links	

-

### PR DESCRIPTION
**Pull request by tkt989**
## Issue
- close #652 

## Overview (Required)
- Add the entire survey to the navigation drawer. At this point, go to https://google.com when you press it.

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: android-base/src/main/java/io/github/droidkaigi/confsched2020/MainActivity.kt [hunk 1]
```diff
@@ -231,6 +231,15 @@ class MainActivity : AppCompatActivity(), HasAndroidInjector {
     private fun handleNavigation(@IdRes itemId: Int): Boolean {
         binding.drawerLayout.closeDrawers()
 
+        when (itemId) {
+            R.id.entire_survey -> {
+                navController.navigate(R.id.chrome, Bundle().apply {
```
Comment (takahirom) [inline android-base/src/main/java/io/github/droidkaigi/confsched2020/MainActivity.kt]: How about adding global action?
https://developer.android.com/guide/navigation/navigation-global-action?hl=en

Comment (tkt989) [reply android-base/src/main/java/io/github/droidkaigi/confsched2020/MainActivity.kt]: @takahirom 
I didn't know that.
This is much better, and I fixed it.
Thanks!

File: android-base/src/main/java/io/github/droidkaigi/confsched2020/MainActivity.kt [hunk 2]
```diff
@@ -231,6 +232,13 @@ class MainActivity : AppCompatActivity(), HasAndroidInjector {
     private fun handleNavigation(@IdRes itemId: Int): Boolean {
         binding.drawerLayout.closeDrawers()
 
+        when (itemId) {
+            R.id.entire_survey -> {
+                navController.navigate(actionGlobalToChrome("https://google.com"))
```
Comment (takahirom) [inline android-base/src/main/java/io/github/droidkaigi/confsched2020/MainActivity.kt]:
```suggestion
                // TODO: Change to the correct URL
                navController.navigate(actionGlobalToChrome("https://google.com"))
```

Comment (tkt989) [reply android-base/src/main/java/io/github/droidkaigi/confsched2020/MainActivity.kt]: I applied your suggestion!:+1:

**Comment by takahirom** [comment]
LGTM!!

**Pull request merged**